pyemma/msm/estimators/maximum_likelihood_hmsm.py

```diff
@@ -195,7 +196,8 @@ def _estimate(self, dtrajs):
                 self.stride = int(min(self.lag, 2*corrtime))
 
         # LAG AND STRIDE DATA
-        dtrajs_lagged_strided = bhmm.lag_observations(dtrajs, self.lag, stride=self.stride)
+        from deeptime.markov import compute_dtrajs_effective
+        dtrajs_lagged_strided = compute_dtrajs_effective(dtrajs, self.lag, n_states=-1, stride=self.stride)
```

Comment:
does this function do the same? `bhmm.lag_observations` is just a plain lag-stride thing, I don't know what effective does here?

Reply:
it does the same but saves me from calling a private function (effective striding is only performed if `stride=='effective'`)